azure-devops/projects/hub-pa-projects/hub-pa-api.tf

```
@@ -1,29 +1,37 @@
-locals {
-  repository_name           = "hub-pa-api"
-  repository_repo_id        = "pagopa/${local.repository_name}"
-  repository_branch_name    = "main"
-  repository_pipelines_path = ".devops"
-  build_path                = "\\${local.repository_name}"
+variable "hub-pa-api" {
+  default = {
+    repository = {
+      organization   = "pagopa"
+      name           = "hub-pa-api"
+      branch_name    = "main"
+      pipelines_path = ".devops"
+    }
+    pipeline = {
+      production_resource_group_name = ""
+      production_app_name            = ""
+      cache_version_id               = "v1"
+    }
+  }
 }
 
 # code review
 resource "azuredevops_build_definition" "hub-pa-api-code-review" {
-  depends_on = [azuredevops_serviceendpoint_github.io-azure-devops-github-ro, azuredevops_project.project]
+  depends_on = [azuredevops_serviceendpoint_github.io-azure-devops-github-pr, azuredevops_project.project]
 
   project_id = azuredevops_project.project.id
-  name       = "code-review"
-  path       = local.build_path
+  name       = "${var.hub-pa-api.repository.name}.code-review"
```

Not blocking ... I will use the built in function format .... it's more terraform > 0.12

---

thanks Walter, do u mean to use this format?

```suggestion
  name       = format("%s.code-review", var.hub-pa-api.repository.name)
```